test/templates/enums/.jhipster/A.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Copy this file to https://github.com/jhipster/generator-jhipster/tree/master/test-integration/samples/.jhipster
With name `FieldTestEnumWithValue.json` or something like this.
Add at:
https://github.com/jhipster/generator-jhipster/blob/30b21eae1e8df878dbd6c21e60e691d54f380fc3/test-integration/scripts/11-generate-entities.sh#L102

And:
https://github.com/jhipster/generator-jhipster/blob/30b21eae1e8df878dbd6c21e60e691d54f380fc3/test-integration/scripts/11-generate-entities.sh#L142

`sql` and `sqlfull` sections.